hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/ProfileServlet.java

```diff
@@ -129,7 +129,7 @@
   private static final int DEFAULT_DURATION_SECONDS = 10;
   private static final AtomicInteger ID_GEN = new AtomicInteger(0);
   static final Path OUTPUT_DIR =
-      Paths.get(System.getProperty("java.io.tmpdir"), "prof-output");
+      Paths.get(System.getProperty("java.io.tmpdir"), "prof-output-ozone");
```

Comment:
Just a question do we need to add some random number to it, so that we don't append output to same file again after restart?

Reply:
no that's not needed. this is the directory name. files names are prepended with the start time, and created in this directory.

Reply:
Thanks @jojochuang for info.